stdlib/SuiteSparse/src/cholmod.jl

```diff
@@ -1,5 +1,13 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+# Theoretically CHOLMOD supports both Int32 and Int64 indices on 64-bit.
```

Comment:
Why delete the comment? These limitations can sometimes to tricky to realize (at least it took me a while) so having them mentioned here might save future developers a bit of frustration.

Reply:
Delete what comment? I've added these.

Reply:
Ha, sorry. That makes more sense. Then I think this is a very good addition to the source.